docs/DeveloperGuide.adoc

```diff
@@ -583,6 +593,74 @@ Mark shows an error message.
 +
 Use case ends.
 
+[#UC11]
+[discrete]
+=== Use case: UC11 - Browse the web
+
+*MSS*
+
+1. User requests to browse the web.
+2. Mark provides in-built web browser.
+3. User browses the web.
++
+Use case ends.
+
+*Extensions*
+[none]
+* 1a. Mark does not have internet access.
+** 1a1. Mark tells user that there is no internet access and suggests user to see cache.
++
+Use case ends.
+
+[#UC12]
+[discrete]
+=== Use case: UC12 - Add bookmarks from web browser
+
+*MSS*
+
+1. User <<UC11,[.underline]#browses the web (UC11)#>>.
```

Comment:
Just a note: the reference UC11 doesn't link to anything at the moment. (I checked the HTML source, and the id of the UC11 header is still "use-case-uc11-browse-the-web".) I think we can fix it in future versions.

Reply:
Clicking on the link works on the developer's guide web page though; which page's html are you referring to?

Reply:
Does the position of the page change? Mine goes to [this page](https://deploy-preview-47--silly-jennings-9dd59a.netlify.com/developerguide#UC11) but the fragment `#UC11` doesn't link to any header.